docs/sphinx/user_guide/feature/reduction.rst

```diff
@@ -159,3 +159,108 @@ Reduction Policies
 For more information about available RAJA reduction policies and guidance
 on which to use with RAJA execution policies, please see 
 :ref:`reducepolicy-label`.
+
+--------------------------------
+Experimental Reduction Interface
+--------------------------------
+
+An experimental reduction interface is now available that hopes to improve
+upon the current reduction model in RAJA. This interface allows ``RAJA::forall``
+to take optional "plugin-like" objects to extend the execution behaviour 
+of a ``RAJA::forall`` execution context.
+
+
+RAJA::expt::Reduce
+..................
+::
+
+  double rs;
+  double rm;
+      
+  RAJA::forall<EXEC_POL> ( Res, Seg, 
+  RAJA::expt::Reduce<RAJA::operators::plus>(&rs),
+  RAJA::expt::Reduce<RAJA::operators::minimum>(&rm),
+  [=] (int i, double& _rs, double& _rm) {
+    _rs += ...
+    _rm = RAJA_MIN(..., _rm); 
+  }
+  );
+  
+  std::cout << rs ...
+  std::cout << rm ...
+
```

Comment:
I want to have the explanation here but I think its more detail than a user needs to think about when they're using the reducers. It would also help to have an example as the "local variable" mentioned here is implicit in the example. I like going into detail about how the reference stuff works, but I think that we should have a more detailed example like an example implementation with all the pieces if we're going to talk about the pieces.

Reply:
I'm not quite sure what you're asking for here

Reply:
@MrBurmark With the most recent changes to the documentation, do you still want this comment to be addressed. I believe the documentation is in a detailed enough state to demonstrate how the reducers work from a users perspective. Is there something specific you want changed / added.